### PR TITLE
Update preinstallNotes for clarity

### PR DESCRIPTION
**Pull request by elingg**

### CODE REVIEW

**Comment by elingg** [comment]
@mgummelt or @gabrielhartmann, please review

File: repo/packages/H/hdfs/1/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
       "url": "https://github.com/mesosphere/hdfs/blob/master/LICENSE"
     }
   ],
-  "preInstallNotes": "In order for HDFS to start successfully, all resources must be available in the cluster, including ports, CPU shares, RAM and disk.\nWe recommend a minimum of five nodes, each with at least 2 CPU shares and 8GB of RAM available for use by the HDFS Service.\nNote that the service is alpha and there may be bugs, including possible data loss, incomplete features, incorrect documentation or other discrepancies.",
+  "preInstallNotes": "In order for HDFS to start successfully, we recommend a minimum of five nodes, each with at least 2 CPU shares and 8GB of RAM available for use by the HDFS Service.\nNote that the service is alpha and there may be bugs, including possible data loss, incomplete features, incorrect documentation or other discrepancies.",
```
Comment (mgummelt) [inline repo/packages/H/hdfs/1/package.json]: "five nodes" isn't accurate, actually.  The nodes must be private.  But we don't expose this well, so I'm not sure if we should chagne it.

**Comment by mgummelt** [comment]
LGTM

File: repo/packages/H/hdfs/1/package.json (hunk 1 above)
Comment (elingg) [inline repo/packages/H/hdfs/1/package.json]: Yep and actually we need a minimum for 4 nodes for our set up but we say 5 so we can have 2 datanodes for replication

**Pull request merged**